articles/hooks/guides/post-change-password.md

```diff
@@ -0,0 +1,131 @@
+
+The parameters listed in the comment at the top of the code indicate the Auth0 objects and their parameters that can be passed into and used by the Hook's function.
+
+```js
+/**
+@param {object} user - The affected user
+@param {string} user.id - user id
+@param {string} user.username - user name
+@param {string} user.email - email
+@param {string} user.last_password_reset - exact date/time the user's password was changed
+@param {object} context - Auth0 connection and other context info
+@param {object} context.connection - information about the Auth0 connection
+@param {object} context.connection.id - connection id
+@param {object} context.connection.name - connection name
+@param {object} context.connection.tenant - connection tenant
+@param {object} context.webtask - webtask context
+@param {function} cb - function (error, response)
+**/
+module.exports = function (user, context, cb) {
+// Perform any asynchronous actions, e.g. send notification to Slack.
+cb();
+};
+```
+
+The callback function `cb` at the end of the sample code is used to signal completion and must not be omitted.
+
+### Response
+
+The Post Change Password Hook ignores any response object.
+
+## Testing Hooks
+
+::: note
+Executing the code using the Runner requires a save, which means that the original code will be overwritten.
+:::
+
+Once you've modified the sample code, you can test Hooks using the Runner. The runner simulates a call to the Hook with the appropriate user information body/payload. The following is the sample body that populates the Runner by default (these are the same objects/parameters detailed in the comment at the top of the sample Hook code):
+
+```json
+{
+  "user": {
+    "tenant": "my-tenant",
+    "username": "user1",
+    "email": "user1@foo.com",
+    "emailVerified": true,
+    "phoneNumber": "1-000-000-0000",
+    "phoneNumberVerified": true,
+    "user_metadata": {
+      "hobby": "surfing"
+    },
+    "app_metadata": {
+      "plan": "full"
+    }
+  },
+  "context": {
+    "requestLanguage": "en-us",
+    "connection": {
+      "id": "con_xxxxxxxxxxxxxxxx",
+      "name": "Username-Password-Authentication",
+      "tenant": "my-tenant"
+    }
+  }
+}
+```
+
+## Example: Use SendGrid to send a password change notification email
+
+```js
+module.exports = function (user, context, cb) {
+
+  const request = require('request');
+
+  // https://sendgrid.api-docs.io/v3.0/mail-send
+  request.post({
+    url: 'https://api.sendgrid.com/v3/mail/send',
+    headers: {
+      'Authorization': 'Bearer YOUR_API_KEY'
+    },
+    json: {
+      personalizations: [{
+        to: [{
+          email: user.email
+        }]
+      }],
+      from: {
+        email: 'admin@example.com'
+      },
+      subject: 'Your password was changed',
+      content: [{
+        type: 'text/plain',
+        value: 'The password for your ' + context.connection.name + ' account ' + user.email + ' was recently changed.'
```

Comment:
```
value: `The password for your ${context.connection.name} account ${user.email} was recently changed.`
```

Reply:
We use `${}` for rendering variables in the docs, so I had to change it to the above to fix errors.